ospf6d/ospf6_top.c

```diff
@@ -878,6 +880,62 @@ DEFUN (no_ospf6_stub_router_admin,
 	return CMD_SUCCESS;
 }
 
+/* Restart OSPF SPF algorithm*/
+static void ospf6_restart_spf(struct ospf6 *ospf6)
+{
+	ospf6_route_remove_all(ospf6->route_table);
```

Comment:
why do all these routes need to be removed - isn't the new limit only used when communicating with zebra? why not just iterate over the routes, look for any where the config change will change what will be sent to zebra? that is, why remove routes and then return them to the state they were in before the config change - that seems disruptive?

Reply:
I thought of doing it that way first but then this will add a lot of complexity to the code and since its a one time CLI and it will be done only when user configures the CLI, hence did it this way.